tst/regression/test_suites/output_hdf5/parthinput.advection

```diff
@@ -64,7 +65,8 @@ file_type = hdf5
 dt = 1.0
 variables = advected, one_minus_advected, & # comments are ok
             one_minus_advected_sq, & # on every (& characters are ok in comments)
-            one_minus_sqrt_one_minus_advected_sq # line
+            one_minus_sqrt_one_minus_advected_sq, & # line
+            metadata_none_var
```

Comment:
Given that this is now in the output file, how about reading the content to make sure that the data written is actually there.
I'm not sure we have that kind of test for `Meshdata::None` already.